pegasus/sites.v3/code.org/views/csf_languages.haml

```diff
@@ -0,0 +1,6 @@
+%h2 
+  %a{href: "/educate/curriculum/elementary-school"} CS Fundamentals (Courses 1-4) is supported in the following languages
+%ul
+  - Languages.get_csf_languages.each do |properties|
+    -if properties[:csf_b] == true
+      %li.bullet= properties[:crowdin_name_s]
```

Comment:
Oops, we forgot to remove `.bullet` also from all of these. That class doesn't exist/was just part of your exercise.